scripts/flash/fill-market-orders.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var async = require("async");
+var chalk = require("chalk");
+var fillOrder = require("../dp/lib/fill-order");
+
+function fillMarketOrderInternal(augur, marketIds, orderType, outcomeId, universe, auth, callback) {
```

Comment:
This `marketIds` parameter doesn't seem to be used, it is immediately overwritten by a callback

Reply:
Copy paste typo. I’ll change to marketId. I’ll make the chamge for clarity